stats/stats.go

```diff
@@ -34,22 +35,27 @@ func (s *Stats) Elapsed() time.Duration {
 	return time.Since(s.start)
 }
 
-func (s *Stats) Print() {
+func (s *Stats) PrintToStderr() error {
 	components := []string{
 		"traversed %d files",
 		"emitted %d files for processing",
 		"formatted %d files (%d changed) in %v",
 		"",
 	}
 
-	fmt.Printf(
+	if _, err := fmt.Fprintf(
+		os.Stderr,
 		strings.Join(components, "\n"),
 		s.Value(Traversed),
 		s.Value(Matched),
 		s.Value(Formatted),
 		s.Value(Changed),
 		s.Elapsed().Round(time.Millisecond),
-	)
+	); err != nil {
+		return fmt.Errorf("failed to write to stderr: %w", err)
+	}
```

Comment:
I would swallow that error, just like fmt.Printf does.

Reason is that if the log is not getting outputted, it doesn't affect the logic of the program.